hazelcast-spring/src/main/resources/hazelcast-spring-4.1.xsd

```diff
@@ -4593,7 +4593,7 @@
             </xs:element>
             <xs:element name="socket-options" type="endpoint-socket-options" minOccurs="0"/>
         </xs:all>
-        <xs:attribute name="name" type="xs:string" use="optional" default=""/>
+        <xs:attribute name="name" type="xs:string" use="optional"/>
```

Comment:
The purpose was to change the one that was about to be released but I got confused because the failing test was using an XM("advancedNetworkConfig-applicationContext-hazelcast.xml") which points to 4.1 xsd. I will update the test and do the change on 4.2 xsd only.